### ISSUE TITLE
Method parameters normal indentation

### ISSUE THREAD

**Issue by blackst0ne** [rubocop-hq/rubocop#6344]
Ruby Style Guide suggests the `normal indent` style: https://github.com/rubocop-hq/ruby-style-guide#no-double-indent

```ruby
# good (normal indent)
def send_mail(source)
  Mailer.deliver(
    to: 'bob@example.com',
    from: 'us@example.com',
    subject: 'Important message',
    body: source.text
  )
end
```

But it seems rubocop doesn't support such a cop yet. 
Am I missing anything or should I open a feature request? 

```ruby
# Actual
def send_mail(source)
  Mailer.deliver(to: 'bob@example.com',
                 from: 'us@example.com',
                 subject: 'Important message',
                 body: source.text)
end
```

```ruby
# Expected
def send_mail(source)
  Mailer.deliver(
    to: 'bob@example.com',
    from: 'us@example.com',
    subject: 'Important message',
    body: source.text
  )
end
```

**Comment by jekuta** [comment]
Correct me if I'm mistaken, but it seems to me that using the [`Layout/FirstMethodArgumentLineBreak`](https://docs.rubocop.org/en/latest/cops_layout/#layoutfirstmethodargumentlinebreak) (it is disabled by default) should enforce this style?
